packages/styles/settings-tools/_s.fonts-faces.scss

```diff
@@ -7,60 +7,62 @@ $font-path: '/fonts';
   }
 }
 
-@font-face {
-  font-family: 'LeroyMerlin';
-  src:
-    url('#{$font-path}/LeroyMerlinSans-Web-Light.woff2') format('woff2'),
-    url('#{$font-path}/LeroyMerlinSans-Web-Light.woff') format('woff');
-  font-weight: 300;
-  font-style: normal;
-}
+@mixin set-font-family() {
```

Comment:
I think `import-font-faces`, or `import-font-families` would be more explicit about the purpose. No need for the `set` prefix, since it's not a property/value type of mixin

Reply:
i'm agree with that ;)